src/darwin/Framework/CHIP/zap-generated/MTRBaseClusters.h

```diff
@@ -23146,6 +23146,12 @@ typedef NS_OPTIONS(uint32_t, MTRApplianceEventsAndAlertAlertStructure) {
     MTRApplianceEventsAndAlertAlertStructurePresenceRecovery = 0x3000,
 };
 
+typedef NS_ENUM(uint8_t, MTRDiscoBallRotate) {
+    MTRDiscoBallRotateOff = 0x00,
```

Comment:
This should not be public API, ideally...

I guess we have this problem with TestCluster too, but we should fix that too...  not sure adding a second set of test things that are exposed is ideal.